components/faucet/CTA.tsx

```diff
@@ -0,0 +1,49 @@
+import { Flex } from "@chakra-ui/react";
+import Link from "next/link";
+import type { FC } from "react";
+import { Button, Text } from "tw-components";
+
+const CTA: FC = () => {
```

Comment:
```suggestion
export const CTA: React.FC = () => {
```

prefer named exports & `React.FC` (no need to separately import `FC` that way)